### DISCUSSION TITLE
What are the supported Intel FPGAs and where are they listed?

### DISCUSSION THREAD

**Discussion by JamieJQuinn** [spcl/dace#1019] [Q&A] [answered]
It's currently unclear which Intel FPGAs are currently supported and the value of `board` to set for each one. Seems like the default is  `a10gx` in [the config](https://github.com/spcl/dace/blob/9ef6de3bf7794dafea2152924ef4cca7561782b9/dace/config_schema.yml#L535). Am I missing where the supported values of `board` are listed? Thanks!

**Comment by TizianoDeMatteis** [comment, marked as answer]
Hi Jamie,

we should be able to generate code for various Intel FPGA boards, as long as they are supported by the Intel FPGA OpenCL Offline Compiler (`aoc`). We tested in hardware for a Nallatech Arria 10 board (`a10gx`) and a Bittware Stratix 10 (`p520_max_sg280l`).


To compile against a particular device, you need to change the `board` configuration entry, indicating an FPGA whose BSP is installed in your system. You can get a list by running `aoc --list-boards`.

Does this answer your question?

Best,
Tiziano

**Reply by JamieJQuinn** [reply]
Yes, that's perfect, thank you!
